op5build/check_k8s.spec

```diff
@@ -35,13 +29,11 @@ source .venv/bin/activate
 python3 -m pip install pytest
 python3 -m pytest
 
-%{__install} -D -p %{exec_path} %{buildroot}/%{plugin_root}/%{exec_path}
-cp --archive %{pkg_path} %{buildroot}/%{plugin_root}/
+%{__install} -D -p %{exec_path} %{buildroot}%{prefix}/%{exec_path}
+cp --archive %{pkg_path} %{buildroot}%{prefix}/
 
 %files
-%defattr(-, monitor, root)
-%attr(755, monitor, root) %{plugin_root}/%{exec_path}
-%attr(755, monitor, root) %{plugin_root}/%{pkg_path}/*
+%{prefix}
```

Comment:
Will the installed command still be executable with this change?

Reply:
Yes, it will use whatever permissions is set on the file in %install. But just to be a little more explicit, I changed the install command to the default for file permissions, which is 755.